Resources/Prototypes/Entities/Objects/Tools/lighters.yml

```
@@ -0,0 +1,51 @@
+﻿- type: entity
+  id: SmallboxItem
+  parent: BaseItem
+  abstract: true
+  components:
+    - type: Storage
+      capacity: 10
+      itemSizeLimit: 1
+    - type: Item
+      size: 6
+
+- type: entity
+  name: Match stick
```

```suggestion
  name: match stick
```

(All names should be lowercased)